great_expectations/dataset/util.py

```diff
@@ -143,6 +154,12 @@ def recursively_convert_to_json_serializable(test_obj):
         ## to the number of digits for which the string representation will equal the float representation
```

Comment:
This comment (not changed here) seems wrong: shouldn't 120/131 have an explicit check on precision?

Reply:
I'm not sure, but this isn't a change in this version.

I propose we open a new issue to look into it.

Reply:
Opened...but then looking at the source, I think there must have been some other merge issue that made my review confusing...it does have the precision check I was expecting...but it currently is only checking a couple of specific numpy types...which we should revisit.